auslib/db.py

```diff
@@ -2490,6 +2491,53 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None, dry
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction, dryrun=dryrun)
 
 
+class EmergencyShutoff(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('emergency_shutoff', metadata,
+                           Column('shutoff_id', Integer, primary_key=True, autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75), nullable=False),
+                           Column('updates_disabled', CompatibleBooleanColumn, default=False, nullable=False),
+                           Column('additional_notification_list', String(500)))
+        AUSTable.__init__(self, db, dialect, scheduled_changes=True, scheduled_changes_kwargs={"conditions": ["time"]})
+
+    def getEmergencyShutoff(self, shutoff_id):
```

Comment:
Is this (and the next) method adding any value? Can callers just call select() on their own?

Reply:
Just to keeping api module more clean, select() can be called directly.